Curricular Models/GenEvo/GenEvo 4 Competition.nlogo

```diff
@@ -10,7 +10,7 @@ breed [ ecolis ecoli ]
 
 ecolis-own [
   my-model              ; cell model (LevelSpace child model) associated with an E. coli cell
-  my-model-path         ; path to locate the file to be loaded as a cell model (LevelSpace child model)
+  my-model-path         ; path to locate the file to be created as a cell model (LevelSpace child model)
```

Comment:
"the file to be created": that is also weird. I'll replace it with "the file to be used for creating".